test/helpers/index.js

```diff
@@ -0,0 +1,36 @@
+import express from 'express';
```

Comment:
I'd like to go for `/test` being a mirror of `/src` with helpers at the top level as well as fixtures for the tests which can be reused. For example in there we should have a file that exports a list of good working torrents as well as a list of broken ones we can use to test that our code doesn't break because of missing fields, etc.

We should probably ditch the generation code and just pop the output from the feeds into files like `testdata` file.